tests/integration/init/schemas/test_init_with_schemas_command.py

```diff
@@ -270,14 +294,17 @@ def test_init_interactive_with_event_bridge_app_aws_schemas_go(self):
             self.assertTrue(expected_output_folder.is_dir())
             self.assertTrue(Path(expected_output_folder, "HelloWorld", "schema").is_dir())
 
+    @pytest.mark.timeout(300)
```

Comment:
Can we set a default in `pytest.ini` file so that we don't need to annotate every test method?

Reply:
Added a global timeout